site/content/docs/5.3/examples/download-app/index.html

```diff
@@ -0,0 +1,282 @@
+  </symbol>
+  <symbol fill="currentColor" viewBox="0 0 1000 1000" id="developer">
+    <path fill-rule="evenodd" d="M657.7 422.409a229.959 229.959 0 0 1-315.39.008A224.953 224.953 0 0 0 225.063 615H225v210a100 100 0 0 0 100 100h450V620a224.944 224.944 0 0 0-117.3-197.591ZM500 435a180 180 0 1 0-180-180 180 180 0 0 0 180 180Zm-29.521 399h.047a12.907 12.907 0 0 0 12.43-9.454l60.006-215.5A12.64 12.64 0 0 0 530.8 593h-.047a13.4 13.4 0 0 0-12.9 9.812L458.048 817.6a12.923 12.923 0 0 0 12.431 16.4ZM306.893 732.747l105.75 60.619a5.191 5.191 0 0 0 6.937-1.657 4.67 4.67 0 0 0 .72-2.484V773.17a10.476 10.476 0 0 0-5.482-9.118l-85-47.369a3.624 3.624 0 0 1-1.4-5.08 3.8 3.8 0 0 1 1.4-1.338l85.622-47.72a9.286 9.286 0 0 0 4.86-8.083v-16.618a5.019 5.019 0 0 0-5.132-4.9 5.315 5.315 0 0 0-2.637.7l-105.659 60.564A13.409 13.409 0 0 0 300 705.8v15.311a13.453 13.453 0 0 0 6.893 11.636Zm281.464 60.619 105.75-60.619A13.453 13.453 0 0 0 701 721.114V705.8a13.409 13.409 0 0 0-6.872-11.6l-105.66-60.565a5.308 5.308 0 0 0-2.635-.7 5.02 5.02 0 0 0-5.133 4.9v16.618a9.286 9.286 0 0 0 4.86 8.083l85.622 47.72a3.8 3.8 0 0 1 1.4 1.338 3.624 3.624 0 0 1-1.4 5.08l-85 47.369a10.476 10.476 0 0 0-5.482 9.117v16.055a4.661 4.661 0 0 0 .721 2.485 5.189 5.189 0 0 0 6.936 1.666Z"/>
+  </symbol>
+  <symbol fill="currentColor" viewBox="0 0 30 30" id="tick">
+    <path d="M26.76 6.205a2.017 2.017 0 00-3.4-1.481h-.001L11.41 16.356l-3.937-4.385a2.01 2.01 0 00-2.742.106l-.808.812a2.038 2.038 0 00-.149 2.704l6.463 8.122a2.012 2.012 0 003.153 0L26.318 7.473a2.028 2.028 0 00.443-1.268z"/>
+  </symbol>
+</svg>
+
+<header class="sticky-top">
+  <nav class="navbar navbar-dark bg-dark navbar-expand-lg" aria-label="Global navigation - With one line title example">
+    <div class="container-xxl">
+      <div class="navbar-brand">
+        <a class="stretched-link" href="#">
+          <img src="/docs/5.3/assets/brand/orange-logo.svg" width="50" height="50" alt="Boosted - Back to Home" loading="lazy">
+        </a>
+        <h1 class="two-lined">
+          Orange Design System
+          <br>
+          Mobile code libraries
+        </h1>
+      </div>
+    </div>
+  </nav>
+</header>
+
+<main>
+  <div class="bg-dark mb-2 border-dark border-bottom border-1 position-relative">
+    <div class="col-12 col-lg-7 out-of-grid h-100 end-0 overflow-hidden">
+      <img loading="lazy" class="h-100 object-fit-cover" alt="App demo_Image PC and app demo_1" src="Banner_image.png">
```

Comment:
Looks like a decorative image, right? Does it  add any information to the content of the page?

If it's a decorative image, 3 possible solutions:
```
alt=""
```
```
aria-hidden="true"
```
```
role="presentation"
```